doc/user/data/self_managed/materialize_operator_chart_parameter.yml

```diff
@@ -165,7 +165,7 @@ parameters:
   - parameter: operator.image.tag
     description: |
       The tag/version of the operator image to be used
-    default: "v0.130.4"
+    default: "v0.138.0"
```

Comment:
Not sure if this refers to envd or orchestratord. I'm assuming orchestratord

Reply:
we just reproduce https://github.com/MaterializeInc/materialize/blob/1520e1583418919f471e76c4af73d6057cb785bb/misc/helm-charts/operator/README.md#parameters